presto-orc/src/test/java/com/facebook/presto/orc/AbstractDwrfStripeCachingTest.java

```diff
@@ -0,0 +1,286 @@
+     * following values:
+     * Column 0: row number
+     * Column 1: Integer.MAX_VALUE
+     * Column 2: row number * 10
+     */
+    private static TempFile writeOrcFile(boolean cacheEnabled, DwrfStripeCacheMode cacheMode, DataSize cacheMaxSize)
+    {
+        TempFile outputFile = new TempFile();
+        try {
+            Type type = INTEGER;
+            List<Type> types = ImmutableList.of(type, type, type);
+            OrcWriterOptions writerOptions = OrcWriterOptions.builder()
+                    .withStripeMaxRowCount(100)
+                    .build();
+            DwrfWriterOptions dwrfWriterOptions = DwrfWriterOptions.builder()
+                    .withStripeCacheEnabled(cacheEnabled)
+                    .withStripeCacheMode(cacheMode)
+                    .withStripeCacheMaxSize(cacheMaxSize)
+                    .build();
+
+            OrcWriter writer = new OrcWriter(
+                    new OutputStreamDataSink(new FileOutputStream(outputFile.getFile())),
+                    ImmutableList.of("Int1", "Int2", "Int3"),
+                    types,
+                    DWRF,
+                    ZLIB,
+                    Optional.empty(),
+                    NO_ENCRYPTION,
+                    writerOptions,
+                    Optional.of(dwrfWriterOptions),
+                    ImmutableMap.of(),
+                    HIVE_STORAGE_TIME_ZONE,
+                    true,
+                    BOTH,
+                    new OrcWriterStats());
+
+            // write 4 stripes with 100 values each
+            int cnt = 0;
+            for (int stripe = 0; stripe < 4; stripe++) {
+                BlockBuilder[] blockBuilders = new BlockBuilder[3];
+                for (int i = 0; i < blockBuilders.length; i++) {
+                    blockBuilders[i] = type.createBlockBuilder(null, 100);
+                }
+
+                for (int row = 0; row < 100; row++) {
+                    blockBuilders[0].writeInt(cnt);
+                    blockBuilders[1].writeInt(Integer.MAX_VALUE);
+                    blockBuilders[2].writeInt(cnt * 10);
+                    cnt++;
+                }
+
+                Block[] blocks = new Block[blockBuilders.length];
+                for (int i = 0; i < blocks.length; i++) {
+                    blocks[i] = blockBuilders[i].build();
+                }
+                writer.write(new Page(blocks));
+            }
+
+            writer.close();
+        }
+        catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+        return outputFile;
+    }
+
+    static DwrfProto.Footer readFileFooter(File orcFile)
+    {
+        try (RandomAccessFile file = new RandomAccessFile(orcFile, "r")) {
+            // read postscript size
+            file.seek(file.length() - 1);
+            int postScriptSize = file.read() & 0xff;
+
+            // read postscript
+            long postScriptPos = file.length() - postScriptSize - 1;
```

Comment:
nit: `postScriptPosition`

Reply:
fixed for this and other variables in this class